Source/Core/DolphinWX/MainMenuBar.cpp

```diff
@@ -545,7 +545,7 @@ void MainMenuBar::RefreshSaveStateMenuLabels() const
   }
 }
 
-void MainMenuBar::RefreshWiiSystemMenuLabel() const
+void MainMenuBar::RefreshWiiToolsLabels() const
```

Comment:
This should likely be left as is and have another function call it along with the new code.

```cpp
void MainMenuBar::RefreshWiiMenuLabels()
{
  RefreshWiiSystemMenuLabel();
  // Your stuff here or another RefreshWhatever() function.
}
```

This keeps unnecessary stuff out of scope in the context of your code.